dev/fattest.simplicity/src/componenttest/containers/ArtifactoryImageNameSubstitutor.java

```diff
@@ -40,6 +43,27 @@ public class ArtifactoryImageNameSubstitutor extends ImageNameSubstitutor {
      */
     private static final String mirror = "wasliberty-docker-remote";
 
+    /**
+     * TODO remove this temp mirror and either build these images at runtime
+     * or pull from a self hosted docker repository
+     *
+     * Artifactory keeps a set of community docker images that have been removed
+     * from DockerHub within this organization specifically for the Liberty builds.
+     */
+    private static final String tempMirror = "wasliberty-infrastructure-docker";
```

Comment:
Technically this is not a mirror so I would prefer if this said `tempRepository` and the variable below said
`tempRepositoryPrefixes`

But do not feel strongly enough to block this PR based on that.